relay-server/src/actors/project_cache.rs

```diff
@@ -467,6 +467,10 @@ struct ProjectCacheBroker {
 }
 
 impl ProjectCacheBroker {
+    /// Hours to wait till the [`ProjectState`] is evictd from the cache if the current incident is
+    /// ongoing, e.g. backoff is initiated and new proejct state cannot be fetched.
+    const INCIDENT_EVICTON_TIMEOUT: u32 = 12;
```

Comment:
This is a personal preference, but I would word this around the specific problem instead of "incident". Since this issue is specific to the network `NETWORK_BACKOFF_EVICTION_TIME` may be more suitable (you may also want to keep it shorter 😄).